testing/internal/e2e/tests/database/migration_test.go

```diff
@@ -150,16 +175,12 @@ func setupEnvironment(t testing.TB, ctx context.Context, c *config) TestEnvironm
 		Boundary:   b,
 		Database:   db,
 		Target:     target,
+		Vault:      v,
 		DbInitInfo: dbInitInfo,
 	}
 }
 
-func populateBoundaryDatabase(t testing.TB, ctx context.Context, c *config, te TestEnvironment) {
-	os.Setenv("BOUNDARY_ADDR", te.Boundary.UriLocalhost)
-	os.Setenv("E2E_PASSWORD_AUTH_METHOD_ID", te.DbInitInfo.AuthMethod.AuthMethodId)
-	os.Setenv("E2E_PASSWORD_ADMIN_LOGIN_NAME", te.DbInitInfo.AuthMethod.LoginName)
-	os.Setenv("E2E_PASSWORD_ADMIN_PASSWORD", te.DbInitInfo.AuthMethod.Password)
-
+func populateBoundaryDatabase(t testing.TB, ctx context.Context, c *config, e TestEnvironment) {
 	// Create resources for target. Uses the local CLI so that these methods can be reused.
 	// While the CLI version used won't necessarily match the controller version, it should be (and is
 	// supposed to be) backwards ompatible
```

Comment:
Can't we pass in this auth information to `AuthenticateAdminCli`?

Reply:
Did some refactoring. Let me know what you think